src/vaadin-login-form.html

```diff
@@ -66,27 +66,22 @@
         static get is() {
           return 'vaadin-login-form';
         }
+
         static get version() {
           return '1.1.0';
         }
 
-        static get properties() {
-          return {
-            /**
-             * Defines the theme of the element.
-             * The value is propagated to vaadin-login-form-wrapper element.
-             */
-            theme: {
-              type: String
-            }
-          };
-        }
-
+        /** @protected */
         connectedCallback() {
           super.connectedCallback();
           this._handleInputKeydown = this._handleInputKeydown.bind(this);
         }
 
+        /**
+         * @param {StampedTemplate} dom
+         * @return {null}
```

Comment:
Note, this is done because of compatibility with `ElementMixin` where this [method is defined](https://github.com/Polymer/polymer/blob/76c71e186ecc605294c3575dd31ac7983a8b3ae3/lib/mixins/element-mixin.d.ts#L221).

While we mark return type as `null`, it's completely optional to actually return anything. 
The only place where the return value is used is [this line](https://github.com/Polymer/polymer/blob/76c71e186ecc605294c3575dd31ac7983a8b3ae3/lib/mixins/element-mixin.js#L714) and we don't use `this.root`.